### PR TITLE
allow any command to be run without using `run` prefix

### PR DESCRIPTION
**Pull request by ghedamat**
should be good

### CODE REVIEW

**Comment by rcy** [comment]
```
rcy@sturgeon:run-anything ~/DEV/PN/pndev$ cat .pndev/foo
#!/usr/bin/env bash

echo hello foo world

false
rcy@sturgeon:run-anything ~/DEV/PN/pndev$ cargo run foo
    Finished dev [unoptimized + debuginfo] target(s) in 0.05s
     Running `target/debug/pndev foo`
catalog_dockerhost_1 is up-to-date
catalog_mailcatcher_1 is up-to-date
catalog_redis_sidekiq_1 is up-to-date
catalog_postgres_1 is up-to-date
catalog_redis_cache_1 is up-to-date
catalog_nginx_1 is up-to-date
hello foo world
Error: nix-shell failed to run command
rcy@sturgeon:run-anything ~/DEV/PN/pndev$ echo $?
1
```

so the Error above is somewhat misleading... it did run the command, but the command exits with 1 (note false in the `foo` script).  Overall it works as expected.

**Comment by rcy** [comment]
I think https://github.com/PrecisionNutrition/pndev/blob/run-anything/src/shell.rs#L215 should show the exit code instead

File: src/shell.rs [hunk 1]
```diff
@@ -212,7 +213,7 @@ pub fn run(cmd: &str) -> Result<ExitStatus, Error> {
     Shell::new()
         .cmd("nix-shell")
         .args(args)
-        .error_msg("nix-shell failed to run command")
+        .error_msg("nix-shell --run failed")
```
Comment (rcy) [inline src/shell.rs:216]: thanks

**Review by rcy — APPROVED** [review]

**Comment by ghedamat** [comment]
bors merge

**Pull request merged**